docker/prod.sh

```diff
@@ -1,17 +1,16 @@
 docker pull jsperf/jsperf.com:master
 docker run -d --name jsperfcom_web_man \
 --link jsperfcom_db_1:db \
---env DOMAIN \
---env SCHEME \
+--env DOMAIN=$DOMAIN \
+--env SCHEME=$SCHEME \
 --env PORT=80 \
---env ADMIN_EMAIL \
---env BROWSERSCOPE \
---env BELL_COOKIE_PASS \
---env COOKIE_PASS \
---env GITHUB_CLIENT_ID \
---env GITHUB_CLIENT_SECRET \
---env MYSQL_PASSWORD \
+--env ADMIN_EMAIL=$ADMIN_EMAIL \
+--env BROWSERSCOPE=$BROWSERSCOPE \
+--env BELL_COOKIE_PASS=$BELL_COOKIE_PASS \
+--env COOKIE_PASS=$COOKIE_PASS \
+--env GITHUB_CLIENT_ID=$GITHUB_CLIENT_ID \
+--env GITHUB_CLIENT_SElsCRET=$GITHUB_CLIENT_SECRET \
```

Comment:
Is this a spelling error? It looks like someone entered 'ls' in the terminal but had the wrong window in focus :)